### ISSUE TITLE
Can't change field properties in form

### ISSUE THREAD

**Issue by alexey-vasyunin** [vitmalina/w2ui#1519]
I can't to change field properties in autogenerated form (didnot try static template).
For example: http://w2ui.com/web/demos/#!forms/forms-10
In console I typed:

> w2ui.form.set("first_name", { type: 'text', required: true, html: { caption: 'ANY NAME', attr: 'style="width: 300px"' } });

geted 'true' in result. No changes at form. Did 

> w2ui.form.refresh();

and no changes also

**Comment by pierpaolocira** [comment]
Yes, I can confirm that only code like the following seems to work:
`w2ui.form.set("first_name", { required: false });`
or
`w2ui.form.set("first_name", { type: 'int' }); `
(or both together) without needing to invoke refresh()

Another interesting thing is that you cange type from text to int, you will keep a textfield allowing only for numbers... BUT if you change from textare to int, you will not have the same (and expected) result: you will have a textarea allowing only for numbers.

Furthermore, when inspecting the field with `w2ui.form.set("first_name")` you can see that all the properties are set to the new values.
